backend/index.js

```diff
@@ -32,6 +32,16 @@ app.set('view engine', 'njk')
 
 nunjucksSetup(app, path)
 
+app.use((req, res, next) => {
```

Comment:
Just so you know we normally merged objects using the spread operator. 
So this:
`bject.assign(options, { phaseName: config.phaseName })`

Becomes:
`    const optionsWithPhaseName = {...options, phaseName: config.phaseName }`

On another note there is established pattern for this sort of thing already, essentially it's a bit of middleware that lives in setUpRoutes.js and places data into local state, namely res.locals.

I'd put the following in setUpRoutes.js
`  router.use(req, res, next) => {
    res.locals = {
      ...res.locals,
       phaseName: config.phaseName 
    }
    next()
  })`

res.locals will be available to the views automatically.

Reply:
how about simply `app.locals.phaseName = config.phaseName` this seems more natural to what needs to be done as the value never changes once the application is running.